libraries/Wire/Wire.h

```diff
@@ -28,9 +28,7 @@
 
 #include "RingBuffer.h"
 
-#define BUFFER_LENGTH 32
```

Comment:
i fixed here in my code as you deleted this line. but another libararies named I2Cdev uses this define. how to do next?